@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/hellfire/fn_setLaserChannels.sqf

```diff
@@ -0,0 +1,62 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_setLaserChannels
+
+Description:
+	Setting of Ace laser code
+
+Parameters:
+    _heli - the helicopter
+    _change - "Primary_Alternate"
+               "LRST_LST"
+               "LaserBoxChannels"
+    _NewLaserchannel - The Alphabetcs channel you wish to set in another variable
+    _indexToChange - The index you want changing
+
+Returns:
+	Returns - "invalid" if it is considered an invalid ace code
+
+Examples:
+    Success = [_heli, "Primary_Alternate", "D", 0] call fza_fnc_setLaserChannels;
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli", "_Request", "_NewLaserchannel","_indexToChange"];
```

Comment:
Once the variable layout has been updated, I don't think this function will be needed.